load-testing/tests/relays_stress.feature

```diff
@@ -5,13 +5,9 @@ Feature: Loading gateway server with relays
     And a rate of "1" relay requests per second is sent per application
```

Comment:
1. Why was the granularity of this test resolved?
2. If (1) is related to us relying on a "persistent chain" is there a way to have two tests:
- High granularity on ephemeral chain
- Low granularity on persistent chain

Reply:
Not sure if I'm understanding this right.

The RPS is not related to the type of chain tested, this number could be changed to any integer we want.

Obviously we can increase the rate to have a higher one, but since we are parsing it as integer we currently can't go below.

We can offer a lower RPS by either:
1. Parse the input number as a float
2. Change the step to something like `a rate of "<int>" relay requests per "<int> seconds is sent per application`

WDYT?